gcs_to_bq/requirements.txt

```diff
@@ -2,24 +2,24 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile requirements.in
+#    pip-compile gcs_to_bq/requirements.in
```

Comment:
Same thing here with the path prefix, I think it's a little confusing. Also a bit annoying that pip-compile generates the comments like this if you run it from another directory.